grid/grid.css

```diff
@@ -0,0 +1,49 @@
+.container{
+    padding: 1rem;
+    gap: 2rem;
+}
+.component-container{
+    padding: 2rem 4rem 2rem 2rem;
+}
+.component-details{
+    padding: 1rem;
+}
+.grid-examples{
+    justify-content: space-evenly;
+    padding: 2rem;
+    flex-wrap: wrap;
+    margin: 1rem;
+    border: 2px solid var(--primary-color);
+    gap: 3rem;
+}
+
+.grid-container{
+    display: grid;
+    column-gap: 1rem;
+    grid-template-columns: auto auto;
```

Comment:
Suggestion: instead of auto i think 50% 50% would be better, but this will also work fine